limap/runners/line_localization.py

```diff
@@ -70,61 +70,49 @@ def get_hloc_keypoints_from_log(logs, query_img_name, ref_sfm=None, resize_scale
 
     return p2ds, p3ds, inliers
 
-def line_localization(cfg, imagecols, linetracks, hloc_log_file, train_ids, query_ids, retrieval, results_path,
-                      coarse_poses=None, img_name_dict=None, ref_sfm=None, resize_scales=None, hloc_name_dict=None, logger=None):
+def line_localization(cfg, imagecols_db, imagecols_query, point_corresp, linemap_db, retrieval, results_path,
+                      img_name_dict=None, logger=None):
     """
     Run visual localization on query images with `imagecols`, it takes 2D-3D point correspondences from HLoc;
     runs line matching using 2D line matcher ("epipolar" for Gao et al. "Pose Refinement with Joint Optimization of Visual Points and Lines");
     calls limap.estimators.pl_estimate_absolute_pose to estimate the absolute camera pose for all query images,
     and writes results in results file in `results_path`.
 
     :param cfg:             dict, configurations which fields refer to `cfgs/localization/default.yaml`
-    :param imagecols:       limap.base.ImageCollection
-    :param linetracks:      iterable of limap.base.LineTrack, LIMAP triangulated/fitted line tracks
-    :param hloc_log_file:   str or Path, path to the log file of HLoc localization, for point correspondences and inlier indices
-    :param train_ids:       iterable of int, image IDs for training/database images
-    :param query_ids:       iterable of int, image IDs for query images
+    :param imagecols_db:    limap.base.ImageCollection of database images
+    :param imagecols_query: limap.base.ImageCollection of query images
```

Comment:
We can add "with uninitialized poses" here, or specify that if the poses are initialized, we can use it as the coarse poses? Or is it how it is working now?